include/deal.II/base/tensor.h

```diff
@@ -1921,6 +1921,47 @@ transpose (const Tensor<2,dim,Number> &t)
 
 
 /**
+ * Return the adjoint of the given tensor.
+ * The adjoint of a tensor $\left(\bullet\right)$ is defined as
+ * @f[
+ *  \textrm{adj}\left(\bullet\right)
+ *   := \textrm{det}\left(\bullet\right) \; \left(\bullet\right)^{-1} \; .
+ * @f]
```

Comment:
Thanks, I've now learned something new (and I now have to unlearn a falsehood): This is actually the [adjugate matrix](https://en.wikipedia.org/wiki/Adjugate_matrix) but, confusingly, is sometimes loosely referred to as the tensor adjoint (probably only by engineers though :-). I'll correct the function name and definition, and I suppose I can then forgo the addition of your note.